workflow/internal/config.schema.yaml

```diff
@@ -0,0 +1,68 @@
+$schema: "https://json-schema.org/draft/2020-12/schema"
+description: "Schema for user-provided configuration files."
+type: object
+required:
+  - download_cutout
+  - backend
+  - layout
+  - aggregate
+additionalProperties: false
+
+properties:
+  download_cutout:
+    type: boolean
+    default: false
+    description: "Whether to download the cutout automatically or use a user-provided one."
+  backend:
```

Comment:
Nothing is actually done with this at the moment and the two `prepare_capacityfactors` scripts are quite hardcoded to the single backend. Do you already have a plan in mind for how to make backend selection work?